plugins/fr.inria.diverse.melange/src/main/java/fr/inria/diverse/melange/Melange.xtext

```diff
@@ -62,19 +62,19 @@ Mapping returns Mapping:
 ;
 
 PackageMapping returns PackageBinding:
-	from=STRING "to" to=STRING "{"
+	from=[ecore::EPackage | QualifiedName] "to" to=STRING "{"
 		(classes+=ClassMapping)*
 	"}"
 ;
 
 ClassMapping returns ClassBinding:
-	from=STRING "to" to=STRING "{"
+	from=[ecore::EClass | QualifiedName] "to" to=STRING "{"
```

Comment:
Actually, we may want to rename EClassifiers (including enums & datatypes). Not a big deal though.